app/scripts/move-popup/dimMovePopup.directive.js

```diff
@@ -95,6 +96,14 @@
 
     vm.stores = dimStoreService.getStores();
 
+    this.canShowAmount = function(item, itemStore, buttonStore) {
+      var result = false;
+      if (item.amount > 1) {
+        result = true;
+      }
+      return result;
+    };
```

Comment:
yepp absolutely. the function has wrong arity too...the moveAmount functionality is not completed and this is poor copy and paste code...shame on me :finnadie: